### ISSUE TITLE
stratisd responds to introspection requests on bogus D-Bus object paths

### ISSUE THREAD

**Issue by mulkieran** [stratis-storage/stratisd#2416]
See https://github.com/stratis-storage/stratisd/pull/2414#issuecomment-785113854 for the behavior on /bogus.

**Comment by mulkieran** [comment]
It seems like dbus-rs 0.8 simply doesn't reject any requests on bogus object paths. Not confirmed with 0.9.

**Comment by mulkieran** [comment]
Related: https://github.com/diwic/dbus-rs/issues/213, because if the object path does not exist, it certainly doesn't support the interface and so an error like this is obtained:
```
[mulhern@localhost ~/stratis2/stratisd (develop-2.3.0)]$ busctl call org.storage.stratis2 /junk org.freedesktop.DBus.ObjectManager GetManagedObjects
Call failed: Method "GetManagedObjects" with signature "" on interface "org.freedesktop.DBus.ObjectManager" doesn't exist
```

whereas with introspect call, you get empty introspection data:

```
[mulhern@localhost ~/stratis2/stratisd (develop-2.3.0)]$ busctl call org.storage.stratis2 /junk org.freedesktop.DBus.Introspectable Introspect
s "<!DOCTYPE node PUBLIC \"-//freedesktop//DTD D-BUS Object Introspection 1.0//EN\"\n\"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd\">\n<node>\n</node>\n"
```
```

**Comment by mulkieran** [comment]
See dbus-rs issue: https://github.com/diwic/dbus-rs/issues/317

**Comment by mulkieran** [comment]
So, it looks like the behavior is expected to change anyway, if we move away from tree data structure. So, I think definitely the think to worry about is that it hangs, and not what message it returns.

**Comment by mulkieran** [comment]
Resolved in current PR #2404:

```
[mulhern@localhost ~/stratis2/stratisd (jbaublitz-issue-stratisd-2277)]$ busctl call org.storage.stratis2 /junk org.freedesktop.DBus.Introspectable Introspect
Call failed: Path, Interface, or Method does not exist
```

**Comment by mulkieran** [comment]
Closed by https://github.com/stratis-storage/stratisd/pull/2439
